syntax_and_semantics/blocks_and_procs.md

```diff
@@ -198,6 +198,18 @@ method &.+(2)
 method &.[index]
 ```
 
+If the block is not the only parameter being passed to the method, the short syntax argument must be supplied within the argument list.
+
+```crystal
+["a", "b"].join(",") { |s| s.upcase }
```

Comment:
I don't think we really need this example, but if we keep it, it should go after the other one which illustrates the actual feature.